app/messaging/process-message.js

```diff
@@ -0,0 +1,41 @@
+const util = require('util')
+const { set } = require('../repositories/application-repository')
+const { applicationResponseMsgType, applicationResponseQueue } = require('../config')
+const sendMessage = require('../messaging/send-message')
+const createReference = require('../lib/create-reference')
+const { notify: { templateIdApplicationComplete } } = require('../config')
+const sendEmail = require('../lib/send-email')
+const boom = require('@hapi/boom')
+const processApplication = async (msg) => {
+  const reference = createReference()
+  await set({
+    reference,
+    type: 'VV001',
```

Comment:
Something for the future perhaps...extract the type code out to a file for 'application-types' (or similar) to make it a bit clearer it is effectively an enum.

Reply:
We will have database lookup rather, understanding new application type can be added at later stage.